src/Messaging/src/RabbitMQ/Host/RabbitHost.cs

```diff
@@ -16,6 +16,9 @@ public sealed class RabbitHost : IHost
         public static IHostBuilder CreateDefaultBuilder() =>
             new RabbitHostBuilder(Microsoft.Extensions.Hosting.Host.CreateDefaultBuilder());
 
+        public static IHostBuilder CreateDefaultBuilder(string[] args) =>
```

Comment:
Is the CodeCov report valid? i.e, does it need tests? Other than that looks good to me.

Reply:
Good call, just added command line arg test